app/src/actions/user.js

```diff
@@ -57,7 +57,7 @@ export function logout() {
       type: LOGOUT
     });
     window.location.hash = window.location.hash.replace(/#access_token=([a-zA-Z0-9.\-_]*)/g, '');
-    if (window.location.pathname.match('^\/map')) {
+    if (window.location.pathname.match('/map')) {
```

Comment:
Hum... searching for why eslint sees the line as wrong, I've found this issue: https://github.com/eslint/eslint/issues/6148

It seems that we may need to escape the slash itself: `window.location.pathname.match('^\\/map')`

Do you know more about that @nerik?